data/language/en-GB.txt

```diff
@@ -25,7 +25,7 @@ STR_0020    :Chairlift
 STR_0021    :Corkscrew Roller Coaster
 STR_0022    :Maze
 STR_0023    :Spiral Slide
-STR_0024    :Go Karts
+STR_0024    :Go-Karts
```

Comment:
What is the reason behind this proposed change? (I did see #10194, but I believe this term is not hyphenated in English.)